app/src/main/java/nl/tudelft/bbw/controller/BlockController.java

```diff
@@ -261,4 +279,39 @@ private boolean verifySignature(byte[] signature, byte[] message) {
         }
     }
 
+    /**
+     * Method to add the multichain (the pairing person database) into your database
+     *
+     * @param multichain given the multichain of the pairing partner
+     * @throws BlockAlreadyExistsException
+     * @throws HashException
+     */
+    public void addMultichain(List<List<Block>> multichain) throws BlockAlreadyExistsException, HashException {
+        if (multichain.isEmpty()) {
```

Comment:
This if statement could be negated, and the rest of the code could be thrown in it.